demos-europe/demosplan-core
# feat(refs DPLAN-12323, #AB15313): add data protection link to statement modal

### Ticket
https://demoseurope.youtrack.cloud/issue/DPLAN-12323/ADO-Issue-15313-Stellungnahmeformular-nach-DSGVO-Vorgabe-anpassen

- Adds data privacy link to the stn dialog
- Aligns the info icon with the info text

![stmnt_modal_box](https://github.com/user-attachments/assets/cb2562d2-a10b-4a97-a909-f1db9fc58ac9)

### How to review/test
- Go to Startseite and click on any listed Verfahren
- Click on "Reden Sie mit"
- You should see Bitte beachten Sie die geltende [Datenschutzerklärung] with a link.
- If you login as private person, you should also see this sentence
- Test being logged in and being logged out.

### Linked PRs
- [ ] https://github.com/demos-europe/demosplan-project-diplanbau/pull/241

### PR Checklist
